### DISCUSSION TITLE
How do you implement absolute paths inside libs/apps?

### DISCUSSION THREAD

**Discussion by Demianeen** [nrwl/nx#21968] [General]
Hello everyone! I would like to use absolute paths in some modules with @nx/vite, but I am not sure where to add new absolute paths. If I add it to global `tsconfig.base`, than other libs autocomplete would suggest those paths as well.

I thought to make those bigger modules buildable and specify absolute paths only there, but then all absolute paths from `tsconfig.base` are overwritten.

How do you implement absolute paths inside libs/apps?

**Comment by Toreckk** [comment]
I'm experiencing a similar issue.

NX is auto-configuring the libs/apps paths in the tsconfig.base.json. However, I've noticed that when I try to implement additional absolute paths in a specific library or app by adding them to the tsconfig.json of that particular library or app, these new paths override the ones from tsconfig.base.json.

**Comment by Marsony** [comment]
Any news?

**Reply by Demianeen** [reply]
From what I got in my research, I think this this a typescript limitation by design. Though I haven't figured exactly why
